tests/cost_report/io/summary/conftest.py

```diff
@@ -0,0 +1,164 @@
+            reinforced_profile=report_type(),
+            infrastructure=SurroundingsInfrastructure(infrastructure_name="TestInfra1"),
+            infrastructure_location_costs=InfrastructureLocationCosts(
+                location=_point,
+                zone_a=i,
+                zone_a_costs=i * 1.1,
+                zone_b=i * 2,
+                zone_b_costs=i * 2.2,
+                surtax=i * 3,
+            ),
+        )
+        _infra_reports.append(_infra_report1)
+        _infra_report2 = deepcopy(_infra_report1)
+        _infra_report2.infrastructure = SurroundingsInfrastructure(
+            infrastructure_name="TestInfra2"
+        )
+        _infra_reports.append(_infra_report2)
+    return _infra_reports
+
+
+def _create_report(
+    report_type: Type[ReinforcementProfileProtocol],
+    available_points: list[PointSurroundings],
+    selected_locations: int,
+) -> MultiLocationProfileCostReport:
+    def _get_layer(material: str, quantity: float) -> MockLayerReport:
+        _layer_report = MockLayerReport()
+        _layer_report.material = material
+        return _layer_report
+
+    _report = MockSummary()
+    _report.obstacle_locations = available_points[0:selected_locations]
+    _report.infra_multilocation_profile_cost_report = _create_infra_reports(
+        report_type, available_points
+    )
+    _required_klei = 2.4 * selected_locations
+    _required_zand = 4.2 * selected_locations
+    _report.cost_per_km = (_required_klei + _required_zand) * 1234
+    _report.cost_per_km_with_surtax = _report.cost_per_km * 1.5
+    _report.profile_cost_report = ProfileCostReport()
+    _report.profile_cost_report.reinforced_profile = report_type()
+    _report.profile_cost_report.layer_cost_reports = [
+        _get_layer("Klei", _required_klei),
+        _get_layer("Zand", _required_zand),
+    ]
+    return _report
+
+
+def get_locations_reinforcements(
+    summary: KoswatSummary, available_locations: list[PointSurroundings]
+) -> list[StrategyLocationReinforcement]:
+    _matrix = []
+    _available_reinforcements = [
+        type(_lp_report.profile_cost_report.reinforced_profile)
+        for _lp_report in summary.locations_profile_report_list
+    ]
+    for _location in available_locations:
+        _a_measures = list(
+            type(_lp_report.profile_cost_report.reinforced_profile)
+            for _lp_report in summary.locations_profile_report_list
+            if _location in _lp_report.obstacle_locations
+        )
+        _selected_measure = _available_reinforcements[-1]
+        if any(_a_measures):
+            _selected_measure = _a_measures[0]
+        _matrix.append(
+            StrategyLocationReinforcement(
+                location=_location,
+                available_measures=_a_measures,
+                selected_measure=_selected_measure,
+            )
+        )
+    return _matrix
+
+
+@pytest.fixture
+def valid_mocked_summary() -> KoswatSummary:
```

Comment:
```suggestion
@pytest.fixture(name="valid_mocked_summary")
def _get_valid_mocked_summary_fixture() -> KoswatSummary:
```

Reply:
What is the advantage doing it this way? It makes finding the references a bit harder.

Reply:
just consistency on how we are doing everywhere else, but you might leave it as it is.